tests/test_metrics_mql_query.py

```diff
@@ -57,6 +57,7 @@
                 },
                 "limit": None,
                 "offset": None,
+                "extrapolate": None,
```

Comment:
Some quick clarifying questions:
Do we have to pass in the `extrapolate` field? I think maybe we should make it so that the passing in the `extrapolate` field is optional, but if it is passed in, it needs to be either true or false

Reply:
Yes, that is correct, the `extrapolate` field is not mandatory. There are tests in the `test_metrics_mql_query.py` file that do not pass `"extrapolate"` and still get serialize appropriately.